scripts/auto-build

```diff
@@ -162,9 +162,9 @@ if ! scripts/make-with-log \
         DISTS_VM="$built_for_vm" \
         DIST_DOM0="$built_for_dom0" \
         BUILD_LOGS_URL="$build_logs" \
-        GIT_URL_$(echo $COMPONENT | tr - _)=$git_url \
+        GIT_URL_${COMPONENT//-/_}="$git_url" \
```

Comment:
Actually... other scripts in scripts/ are #!/bin/bash and also use `${...//.../...}`.
@marmarek which to prefer?